example/index.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+const path = require("path");
+const express = require("express");
+const complate = require("complate-express");
+
+const bundle = require(path.resolve(__dirname, "dist", "bundle.js"));
+const app = express();
+
+app.use(complate(bundle));
+
+app.get("/", (req, res) => {
+	return res.render("site-index", { title: "Hello World!" });
+});
+
+app.listen(5000);
```

Comment:
The thing I dislike on that complate-express-example is that it is not attached to the actual code, and it does not necessarily evolve with the given code. Providing it in the example section and even further making it a test artifact ensures that it stays in synch.